src/fv/mpfao/tdympfao.c

```diff
@@ -408,7 +436,7 @@ PetscErrorCode TDyCreate_MPFAO(void **context) {
 
   // Initialize defaults and data.
   mpfao->gmatrix_method = MPFAO_GMATRIX_DEFAULT;
-  mpfao->bc_type = DIRICHLET_BC;
+  mpfao->bc_type = NEUMANN_BC;
```

Comment:
I think this change in the default setting might also break a lot of tests.

Reply:
This change was also made to try and match results with PFLOTRAN and can be reverted.